NEWS.md

```diff
@@ -1,3 +1,10 @@
+## 3.1.0 2020-06-15
+
+ * [OKAPI-852](https://issues.folio.org/browse/OKAPI-852) Support PATCH request
+ * [OKAPI-847](https://issues.folio.org/browse/OKAPI-847) Conditionally defer CORS handling to module when invoked via passthrough API
+ * [OKAPI-792](https://issues.folio.org/browse/OKAPI-792) PostgreSQL SSL CA Certificate configuration option
+ * [OKAPI-787](https://issues.folio.org/browse/OKAPI-787) Support SSL connections to Postgres
```

Comment:
Deprecation of postgres_user should be mentioned: https://issues.folio.org/browse/OKAPI-854